packages/Tokens/src/tokens.yaml

```diff
@@ -203,8 +203,8 @@ counter:
   medium: 19px
 #
 pill:
-  small-radius: 8px / 50%
-  medium-radius: 12px / 50%
+  small-radius: calc(8px / 50%)
+  medium-radius: calc(12px / 50%)
```

Comment:
I just realized that SCSS compiler thinks that it is division. But actually it is just a syntax for `border-radius`. I found a workaround here: https://github.com/sass/sass/issues/667

I think we should apply this workaround instead of `calc(...)`

Reply:
Side note: It is the main reason why using `/` for division is deprecated in Dart Sass compiler - because it is hard to differentiate division and CSS syntax.